README.md

```diff
@@ -7,10 +7,66 @@ Travix UI Components' repository.
 ## UI-Kit
 ### How to install and setup
 - `npm i travix-ui-kit -S` install as a dependency
-- add npm script `build` with value `cd node_modules/travix-ui-kit/ && npm run build && cd ../..` to your `package.json`
 
-### How to use
-#### JS
+### Usage
+
+#### CLI
+
+The UI Kit comes with a CLI tool to help you build your UI bundles (JS and CSS).
+
+To see the options available:
+
+```bash
+$ node_modules/.bin/travix-ui-kit -h
+
+  Usage: travix-ui-kit [options]
+
+  Options:
+
+    -h, --help                 output usage information
+    -V, --version              output the version number
```

Comment:
It comes from `commander`: https://github.com/tj/commander.js/issues/560

Nevertheless it is also more common to do `-v` to add verbosity on node CLI tools. Still, it is possible to provide an override of the help I think.